covfee/client/models/Journey.ts

```diff
@@ -85,3 +85,8 @@ export const submitJourney = (id: string) => {
   }
   return fetcher(url, requestOptions).then(throwBadResponse)
 }
+
+export const fetch_annotator_data = (id: string) => {
+  const url = Constants.api_url + "/journeys/" + id + "/annotator_data"
```

Comment:
perhaps simplify the API into `const url = Constants.api_url + "/journeys/" + id + "/annotator"` . I don't know how informative the `_data` is

Reply:
Fixed in 9fbd57d